pkg/bgpv1/manager/reconcile.go

```diff
@@ -420,29 +426,61 @@ func (r *LBServiceReconciler) lbServiceReconciler(
 		existingSelector = sc.Config.ServiceSelector
 	}
 
+	// Populate local availability used for externalTrafficPolicy=local handling
+	sla := make(svcLocalAvailability)
+endpointsLoop:
+	for _, eps := range r.epDiffStore.List() {
+		svcID := eps.ServiceID
```

Comment:
Would it make sense to skip this processing for services that do not have `externalTrafficPolicy=Local` here to save some processing, or is it not worth retrieving the service details at this place?

Reply:
Yep, I think that makes sense. It should be a single map lookup, so worth to pay that cost I guess.

Reply:
Done